File: Ravi_Playing_Game.py
from operator import le
from re import S


def solution(score, queries):
    tot = []
    tot.append(score[0])
    for i in range(1, len(score)):
        tot.append(tot[i-1] + score[i])

    for q in queries:
        print(tot[q-1])


t = int(input(""))
sArr = []

qArr = []


for i in range(0, t):
    sArr.append(int(input("")))


f = int(input(""))
for i in range(0, f):
    qArr.append(int(input("")))


solution(sArr, qArr)

File: Ravi_Playing_Game.cpp
#include <bits/stdc++.h>

using namespace std;

void solution(int *scores, int *queries)
{
    unordered_map<int, int> tot = {};

    int scoresSize = sizeof(scores) / sizeof(scores[0]);
    int queriesSize = sizeof(queries) / sizeof(queries[0]);

    for (int j = 0; j <= scoresSize; j++)
    {
        tot[j] = tot[j - 1] + scores[j];
    }

    for (int j = 0; j < queriesSize; j++)
        cout << tot[queries[j] - 1] << endl;
}

int main()
{
    int t, q;
    int scores[100];
    int queries[int(1e5 + 10)];
    cin >> t;

    for (int i = 0; i < t; i++)
        cin >> scores[i];
    cin >> q;
    for (int i = 0; i < q; i++)
        cin >> queries[i];

    // for (int i = 0; i < q; i++)
    //     cout << queries[i] << " : ";
    // cout << "\n\n\n\n\n\n";
    solution(scores, queries);

    return 0;
}